collective/pdfpeek/browser/utils.py

```diff
@@ -29,6 +31,9 @@ class pdfpeekUtilView(BrowserView):
     def process_conversion_queue(self):
         """process the queue.
         """
+        token = createToken()
+        self.request.set('_authenticator', token)
+
```

Comment:
as i am not into that CSRF stuff i wasn't sure if disabling any security feature is a good thing and worked out that solution without changing any protection mechanisms, or touching it ;)